packages/shared/src/contracts.ts

```diff
@@ -19,13 +14,13 @@ export type TransactionOptions = {
   gasPrice?: bigint;
   maxFeePerGas?: bigint;
   maxPriorityFeePerGas?: bigint;
-  nonce?: bigint;
+  nonce?: number;
```

Comment:
changed this to match the type of ethers. i assume they put some thought into this and figured that number is good enough here 😇